wrapper/src/main/java/com/splunk/support/lambda/ExportersInitializer.java

```diff
@@ -51,21 +58,18 @@ static SdkTracerProvider configureExporters(List<String> exporters, Properties c
         spanProcessors.add(exporter);
       }
     }
-    TraceConfig traceConfig = TraceConfig.builder().setSampler(Sampler.alwaysOn()).build();
 
-    SdkTracerProvider result =
-        SdkTracerProvider.builder()
-            .addSpanProcessor(SpanProcessor.composite(spanProcessors))
-            .setTraceConfig(traceConfig)
-            .build();
-    return result;
+    return SdkTracerProvider.builder()
+        .addSpanProcessor(SpanProcessor.composite(spanProcessors))
+        .setSampler(Sampler.alwaysOn())
+        .build();
   }
 
   private static SpanProcessor createExporter(String exporterName, Properties config) {
```

Comment:
could we use the autoconfigure artifact, rather than repeating all this logic from it?

Reply:
@jkwatson I have this on my plate (I recall our previous discussion about that). Filed an issue https://github.com/signalfx/splunk-otel-java-lambda/issues/28 to implement once SDK reaches stability (1.0)

Reply:
Ah heck, went for autoconf. Please have a look :)